lib/assets/notification.css

```diff
@@ -151,3 +151,10 @@ html, body {
   background: #147EFB;
   color: #fff;
 }
+.flat, .flat:before,
+#notification.flat, #notification.flat:before,
+#notification #buttons.flat, #notification #buttons.flat:before,
+#notification .icon.flat, #notification .icon.flat:before,
+#notification .message.flat, #notification .message.flat:before  {
+  border-radius: 0px;
```

Comment:
I tried that, but the CSS was being clobbered by subsequent more specific selectors it seems. Give it a go. I'm no CSS expert.